core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/LockCache.java

```diff
@@ -95,32 +97,29 @@ private void throwCachedLockException(LockException e) throws LockException
     static final class LockKey
     {
         private final String myDataCenter;
-        private final String myResource;
+        private final String myResourceName;
 
         LockKey(String dataCenter, String resource)
         {
+            Preconditions.checkNotNull(resource);
             myDataCenter = dataCenter;
-            myResource = resource;
+            myResourceName = resource;
```

Comment:
checkNotNull also returns the value it checks... So you could do it on the same line as the assignment (with static import it gets kinda nice ;)
myResourceName = checkNotNull(resource);

Reply:
Thanks for the tip :)